vellum/workflows/triage_message/workflow.py

```diff
@@ -27,6 +29,8 @@ class TriageMessageWorkflow(BaseWorkflow):
                 ParseFunctionCallNode.Ports.email_initiate >> EmailInitiateNode,
                 ParseFunctionCallNode.Ports.text_reply >> TextReplyNode,
                 ParseFunctionCallNode.Ports.slack_reply >> SlackReplyNode,
+                ParseFunctionCallNode.Ports.job_opportunity_recruiter_response >> JobOpportunityRecruiterResponseNode,
+                ParseFunctionCallNode.Ports.job_opportunity_forwarder_confirmation >> JobOpportunityForwarderConfirmationNode,
```

Comment:
This is going to call these two functions / nodes in parallel. Ideally, they are part of the same chain